command/server/config_telemetry_test.go

```diff
@@ -0,0 +1,41 @@
+package server
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMetricFilterConfigs(t *testing.T) {
+	t.Parallel()
+	cases := []struct {
+		configFile            string
+		expectedFilterDefault *bool
+		expectedPrefixFilter  []string
+	}{
+		{
+			"./test-fixtures/telemetry/valid_prefix_filter.hcl",
```

Comment:
Forgot to add fixtures?

Reply:
hah yep just noticed that I didn't push them, looks like the dir I created them in got gitignored, fixing that

Reply:
6c2ad99